Projects/Feature/OutingApplyFeature/Sources/Component/OutingApplicationView.swift

```diff
@@ -2,27 +2,35 @@ import DesignSystem
 import SwiftUI
 
 struct OutingApplyTextField: View {
-    var title: String
+    @State var title: String
     var placeholder: String
     @Binding var text: String
+    var status: Bool?
 
     public init(
         _ title: String = "",
         _ placeholder: String = "",
-        _ text: Binding<String>
+        _ text: Binding<String>,
+        _ status: Bool? = false
     ) {
         self.title = title
         self.placeholder = placeholder
         _text = text
+        self.status = status
     }
 
     var body: some View {
         VStack(alignment: .leading) {
-            Text(title)
-                .dmsFont(.body(.body2), color: .GrayScale.gray10)
-                .padding(.leading, 1)
-                .padding(.bottom, 8)
-
+            HStack {
+                Text(title)
+                    .dmsFont(.body(.body2), color: .GrayScale.gray10)
+                    .padding(.leading, 1)
+                    .padding(.bottom, 8)
+                if self.status ?? false {
+                    Text("*")
+                        .dmsFont(.body(.body3), color: .PrimaryVariant.primary)
+                }
+            }
             DMSFormTextField(placeholder, text: $text)
```

Comment:
```suggestion
                    .padding(.bottom, 8)

                if self.status ?? false {
                    Text("*")
                        .dmsFont(.body(.body3), color: .PrimaryVariant.primary)
                }
            }

            DMSFormTextField(placeholder, text: $text)
```
스타일 가이드 바뀐지는 잘 모르긴하지만 스유 스타일은 이런 스타일로 가던것으로 기억해요